metals/src/main/scala/scala/meta/internal/metals/ScalafixProvider.scala

```diff
@@ -115,6 +115,23 @@ case class ScalafixProvider(
       }
     }
   }
+  private def scalafixSucceded(evaluation: ScalafixEvaluation): Boolean =
+    evaluation.isSuccessful && evaluation
+      .getFileEvaluations()
+      .headOption
+      .forall(_.isSuccessful)
+
+  private def getMessageErrorFromScalafix(
+      evaluation: ScalafixEvaluation
+  ): Option[String] = {
+    if (!evaluation.isSuccessful)
+      evaluation.getErrorMessage().asScala
+    else
+      evaluation
+        .getFileEvaluations()
+        .headOption
+        .flatMap(_.getErrorMessage.asScala)
```

Comment:
If what I have above is correct and we can rely on this always returning something, then we can simplify and just do
```suggestion
      evaluation
        .getFileEvaluations()
        .map(_.getErrorMessage.asScala)
```

But again, only if in reality if a file fails there will always be a message.

Reply:
Yes. it's true. I m removing the default message

Reply:
Honestly, I would keep it all as options, since we cannot assume that it's always correct are that there will never be a regression. 

Having the default message does not cost us anything and just keeps us safe.

Reply:
I agree, it's safer to keep this option. As you like.

Reply:
@ckipp01 what do you think?

Reply:
Yea it's fine. Originally it was just a question on my end.